app/views/shared/components/_case_page_original_case_details.html.slim

```diff
@@ -1,4 +1,4 @@
-- if @case.ico?
+- if @case.ico? || @case.overturned_ico?
```

Comment:
We're using different styles of detecting whether to render bits in views. In `edit.html.slim` we use:

```
  - if @case.respond_to?(:ico_decision) && @case.ico_decision.present?
    ...
```

Which is more in-line with the duck-typing philosophy in Ruby. Feels like we should choose one style and stick with it, what do you think? If we wanted we could use duck-typing here with:

```
  - if @case.respond_to?(:original_case) && @case.original_case.present?
    ...
```

Reply:
I agree with you, I completely forgot we could inspect original_case for both case types.